### PR TITLE
Add booked services field to bookings query

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: src/booking/types/outputs/BookedService.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export default new GraphQLObjectType({
+  name: 'BookedService',
+  fields: {
+    category: {
+      type: GraphQLString,
+      description: 'Type of booked service',
+      resolve: ({ category }: BookedServiceType) => category,
+    },
+    status: {
+      type: GraphQLString,
```
Comment (mrtnzlml) [inline src/booking/types/outputs/BookedService.js]: Can we use enum here? It simplifies implementation of Relay clients (typed clients in general).

Comment (tbergq) [reply src/booking/types/outputs/BookedService.js]: Sure, need to know the statuses, I will see what I can find out

File: src/booking/types/outputs/BookingInterface.js [hunk 2]
```diff
@@ -6,19 +6,26 @@ import {
...
 import GraphQLBookingStatus from '../enums/BookingStatus';
 import Price from '../../../common/types/outputs/Price';
 import BookingDestinationImageURL from '../../resolvers/BookingDestinationImageURL';
+import GraphQLBookedServices from './BookedService';
 
 export type BookingInterfaceData = BookingsItem;
 
+const { connectionType: BookedServicesConnection } = connectionDefinitions({
+  nodeType: GraphQLBookedServices,
+});
```
Comment (mrtnzlml) [inline src/booking/types/outputs/BookingInterface.js]: Do we really need the connection here? What about a simple list? I don't think we are ever going to paginate our services.

File: src/booking/types/outputs/BookedService.js (hunk 1 above)
Comment (mrtnzlml) [reply src/booking/types/outputs/BookedService.js]: Don't worry. I believe enum can be extended without BC breaks and Flow actually forces you to write your clients to be compatible with this change.

File: src/booking/types/outputs/BookedService.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+type BookedServiceType = {|
+  category: string,
+  status: string,
+|};
+
+export default new GraphQLObjectType({
+  name: 'BookedService',
+  fields: {
+    category: {
```
Comment (mrtnzlml) [inline src/booking/types/outputs/BookedService.js]: "Category" also sounds enum-ish. What is it actually? Do you have an example of possible values?

Comment (tbergq) [reply src/booking/types/outputs/BookedService.js]: I wish I did, but I will need it sooner or later, so I might as well find out now

**Comment by tbergq** [comment]
@mrtnzlml please check again

**Pull request merged**